### ISSUE TITLE
lesson 2 practice with a kaggle dataset

### ISSUE THREAD

**Issue by datalass1** [datalass1/fastai#8]
To make sense of learning rates, augmentation, stochastic gradient descent with restarts, precompute=True/False, freeze and unfreeze, epochs, over-fitting, training time augmentation (TTA), confusion matrix, cycle_len and cycle_mult. 

Also:
Exercise: look at the other cases and convince yourself that this make sense.
Exercise: how would you rewrite binary_loss using if instead of * and + ?

**Comment by datalass1** [comment]
log loss: http://wiki.fast.ai/index.php/Log_Loss
uploaded notebook to github.
